CMakeLists.txt

```diff
@@ -304,7 +304,7 @@ option(ENABLE_EXAMPLES "Build examples. Defaults to ON" ON)
 
 option(ENABLE_RDMA "Enable usage of RDMA libraries for memory allocations. Requires libRDMAcm and libverbs" OFF)
 
-option(ENABLE_TRAFFIC_MONITOR "Enable network traffic monitoring & logging in remote device" OFF)
+option(ENABLE_TRAFFIC_MONITOR "Enable network traffic monitoring & logging in remote device" ${ENABLE_REMOTE_CLIENT})
```

Comment:
What's the reason making this optional? The data is accumulated unconditionally so there should not be additional runtime overheads here.

Reply:
The code was originally written as an optional thing and spawns an extra thread for these stats. I've kept the option to turn it off if so desired.

Reply:
But does it impact the overhead. It's not disabling the collection of the status from what I see? I prefer not to have options that are not clearly useful as they clutter the code and also make it more difficult test the different (build) paths.